files/KoreBuild/KoreBuild.Common.props

```diff
@@ -33,17 +33,19 @@ Default layout and configuration.
     <Configuration Condition="'$(CI)' == 'true'">Release</Configuration>
     <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
     <BuildProperties>$(BuildProperties);Configuration=$(Configuration)</BuildProperties>
-    <RepositoryRoot Condition="'$(RepositoryRoot)' == ''">$(MSBuildStartupDirectory)</RepositoryRoot>
-    <RepositoryRoot>$([MSBuild]::NormalizeDirectory('$(RepositoryRoot)'))</RepositoryRoot>
-    <ArtifactsDir>$([MSBuild]::NormalizeDirectory('$(RepositoryRoot)'))artifacts\</ArtifactsDir>
+    <RepoRoot Condition="'$(RepoRoot)' == ''">$(MSBuildStartupDirectory)</RepoRoot>
+    <RepoRoot>$([MSBuild]::NormalizeDirectory('$(RepoRoot)'))</RepoRoot>
+    <!-- Older targets using KoreBuild use RepositoryRoot. Update to use RepoRoot whenever possible as a part of migrating to Arcade. -->
```

Comment:
"Older targets" where exactly? In this repo still or in aspnet/AspNetCore for a very short time?

Suggest adding some more info to this comment unless the older targets won't exist in a couple of days.

Reply:
This project is still open source so anyone could be using KoreBuild. At least a few teams at Microsoft are still using KoreBuild and I don't want to completely break them. Eventually we will completely retire KoreBuild because we only care about supporting the AspNetCore repo.

Reply:
This project is still open source so anyone could be using KoreBuild. At least a few teams at Microsoft are still using KoreBuild and I don't want to completely break them. Eventually we will completely retire KoreBuild because we only care about supporting the AspNetCore repo.